best-practices/README.md

```diff
@@ -218,7 +218,7 @@ Sass
 Browsers
 --------
 
-* Don't support versions of Internet Explorer before IE10.
+* Avoid supporting versions of Internet Explorer before IE10.
```

Comment:
Yep, I like the clarity in your wording. That is essentially what I was trying to get it. It just _depends_.